src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.css

```diff
@@ -2,16 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-.breakpoint-panel-wrapper {
-  min-width: 300px;
-}
-
 .breakpoint-panel {
-  margin: 4px 0;
+  margin: 0.25rem 0 0.5rem -1px;
   background: var(--theme-toolbar-background);
-  border: 1px solid var(--theme-splitter-color);
-  border-left: none;
-  border-right: none;
+  border: 1px solid var(--theme-toolbar-background);
```

Comment:
We add the same colored border as the background to align to other elements like the active paused line which uses borders.